brokers/brokers.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package brokers
```

Comment:
Can we rename this package and (directory) to `pubsub` or something similar, more self-explanatory?